server-common/src/main/java/org/figuramc/figura/server/FiguraPermissions.java

```diff
@@ -1,12 +1,19 @@
 package org.figuramc.figura.server;
 
 import org.figuramc.figura.server.utils.Pair;
+import static org.figuramc.figura.server.FiguraPermissionNodes.*;
 
 import java.util.List;
 
 public class FiguraPermissions {
-    public static final List<Pair<String, Boolean>> PERMISSIONS_LIST = List.of(
-            Pair.of("figura.avatars.immortalize", false),
-            Pair.of("figura.avatars.set", false)
+    public static final List<Pair<FiguraPermissionNodes, Boolean>> PERMISSIONS_LIST = List.of(
+            Pair.of(FIGURA_AVATARS_IMMORTALIZE, false),
+            Pair.of(FIGURA_AVATARS_SET, false)
```

Comment:
could maybe put `FIGURA_AVATARS_CLEAR` in here while you're at it, it's currently inaccessible in MinecraftForge